static/scss/partials/landing/a.scss

```diff
@@ -63,3 +80,216 @@
+        grid-template-columns: repeat(5, 1fr);
+        column-gap: $spacing-2xl;
+    }
+
+    .c-pricing-options {
+        display: grid;
+        grid-column-start: span 3;
+        column-gap: $spacing-md;
+        row-gap: $spacing-md;
+        grid-template-columns: 1fr;
+
+        @media screen and ($mq-sm){
+            padding: 0 $spacing-xl;
+
+            @media screen and ($mq-md){
+                grid-template-columns: 1fr 1fr;
+                row-gap: none;
+                padding: 0;
+            }
+        }
+
+        .c-option {
+            display: flex;
+            flex-direction: column;
+            justify-content: space-between;
+            background: white;
+            align-items: center;
+            border-radius: $border-radius-md;
+            text-align: center;
+            padding: $spacing-xl;
+
+            .c-price {
+                color: $color-blue-50;
+                font-weight: 500;
+                font-size: 1.125em;
+                margin: 0;
+                padding-top: $spacing-md;
+
+                .c-currency {
+                    color: $color-dark-gray-50;
+                    font-size: .55em;
+                    font-weight: normal;
+                }
+
+                .premium-price {
+                    font-size: 1.75em;
+                }
+            }
+
+            .c-features {
+                padding-top: $spacing-md;
+                text-align: left;
+
+                li {
+                    padding-bottom: $spacing-md;
+                }
+
+                li::before {
+                    content: "";
+                    width: 20px;
+                    height: 20px;
+                    margin-top: 6px;
+                    margin-left: -$spacing-lg;
+                    position: absolute;
+                    background-repeat: no-repeat;
+                    background-image: url("/static/images/icon-blue-check.svg");
+                }
+            }
+
+            .c-description {
+                font-size: .875em;
+                border-radius: $border-radius-md;
+                border: 2px solid $color-light-gray-40;
+                padding: $spacing-sm $spacing-md;
+            }
+
+            .mzp-c-button {
+                margin-top: $spacing-lg;
+                float: bottom;
+            }
+        }
+
+    }
+
+    .c-pricing-text {
+        grid-row-start: 1;
+        grid-column-start: span 3;
+        height: 100%;
+        text-align: center;
+
+        h2 {
+            font-family: $font-metropolis;
+            font-weight: 500;
+        }
+
+        .c-text {
+            padding: 0 $spacing-lg;
+        }
+
+        @media screen and ($mq-lg) {
+            text-align: start;
+            grid-column-start: span 2;
+            grid-row-start: auto;
+
+            .c-text {
+                padding: 0;
+                top: 50%;
+                -ms-transform: translateY(-50%);
+                transform: translateY(50%);
```

Comment:
The `top` doesn't do anything here I think (since it's not a [positioned element](https://developer.mozilla.org/en-US/docs/Web/CSS/position#types_of_positioning)). And I think I'd avoid taking the element out of the normal document flow to make it easier to understand how it relates to other elements.

Instead, you can alter the flow to vertically center the element using flexbox (`display: flex`) in a vertical direction (`flex-direction: column`), then centering the contents (`justify-content: center`) and making it as high as its parent element (`height: 100%`).